humanguide/index.rst

```diff
@@ -1,7 +1,58 @@
 Human interface guidelines
 ==========================
 
-Ubuntu Touch follows a clean, beautiful, modern and (above all) convergent design. We aim to keep developing the design started by Canonical.
+Ubuntu Touch follows a modern and convergent design philosophy called Suru.
+We aim to keep developing the design started by Canonical.
+
+.. figure:: /_static/images/humanguide/750w_DesignPhilosophy_MainImage.png
+
+
+Suru meaning
+------------
+
+Here at Ubports we believe that everyone should have access to free, reliable, and trusted software that can be shared and developed by anyone. This has paved the way to allow the community to grow and prosper with freedom trust and collaboration. This integral belief has been translated into our design philosophy called Suru.
+
+- Suru items from the UTouch values alluding to Japanese culture.
+
+- The design of Suru is inspired by origami, because it give us a solid and tangible foundation.
+
+- Paper can be used in all areas of the brand in two and three dimensional forms, because it is transferable and diverse.
+
+Translated in design
+--------------------
+
+Suru brings a precise yet organic structure to the UTouch interface. The sharp lines and varying levels of transparency evoke the edges and texture of paper. All elements are placed deliberately, with the express aim of being easy for the user to identify and use.
+
+When using a small layout, the information and functionality is folded into a compact object that that can be refolded to expose different areas. As the layout size increases, the object can become progressively larger, allowing more of the information and functionality to be exposed at any one time.
+
+Origami
+-------
+
+.. figure:: /_static/images/humanguide/366w_DesignPhilosophy_origami.png
+
+Origami has long been associated with good fortune and represents the visual style for the Ubuntu Phone. Origami folds are used to define the design.
```

Comment:
"Ubuntu Phone" is somehow never used anywhere anymore, let's make this:

"the visual style for Ubuntu Touch."